scripts/check-no-replaces.sh

```
@@ -0,0 +1,20 @@
+# tagging the lockstep release.
+#
+# Invoked only when the workflow runs in "release-prep" mode (controlled by a
+# CI env var or a manual workflow_dispatch). Dev branches keep their replaces.
+
+set -euo pipefail
+
+matches=$(grep -nH '^replace github.com/gruntwork-io/terratest' modules/*/go.mod cmd/*/go.mod 2>/dev/null || true)
+
+if [ -n "$matches" ]; then
+  echo "::error::Local terratest replace directives present in release commit:"
+  echo "$matches" | sed 's/^/    /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)